src/storage/paths.ts

```diff
@@ -26,7 +26,7 @@ export function getSiteThumbnailPath( siteId: string ): string {
 
 export function getResourcesPath(): string {
 	if ( process.env.NODE_ENV === 'development' || process.env.NODE_ENV === 'test' ) {
-		return process.cwd();
+		return app.getAppPath();
```

Comment:
This is only tangentially related to the core functionality of this PR. I made the change because there's no guarantee (that I'm aware of) that the `cwd` is the root folder of the app code, whereas the `app.getAppPath()` return value is normalized by Electron internals.